yamllint/cli.py

```diff
@@ -84,17 +85,31 @@ def standard_color(problem, filename):
             line += '  \033[2m(%s)\033[0m' % problem.rule
         return line
 
+    @staticmethod
+    def json(problem, filename):
+        return json.dumps({
+            "path": filename,
+            "line": problem.line,
+            "char": problem.column,
+            "description": problem.message,
+            "code": "yamllint",
+            "name": "yamllint",
+            "severity": problem.level,
+        })
 
 def show_problems(problems, file, args_format, no_warn):
     max_level = 0
     first = True
+    problems_json = []
 
     for problem in problems:
         max_level = max(max_level, PROBLEM_LEVELS[problem.level])
         if no_warn and (problem.level != 'error'):
             continue
         if args_format == 'parsable':
             print(Format.parsable(problem, file))
+        elif args_format == 'json':
+            problems_json.append(json.loads(Format.json(problem, file)))
```

Comment:
The code does `json.dumps()` then `json.loads()` then `json.dumps()` again. Maybe it could be simplified.

Reply:
Thx for taking the time. All the above are valid, maybe i was a bit hasty or this one got me too many headaches. (not a python pro...yet;)), and yes it's ugly.

But i am not sure how to make python return the json-object instead of the json-string and NOT break json-validity (there are descriptions containing double-quotes).

Reply:
No problem, your code looks globally good!

The first thing you could do is remove `json.loads()` from here and `json.dumps()` from `Format`. It would change anything, except lightening the code.

Another approach would be to directly print json.dumped strings + `',\n'` in this line, and handle the starting `[` and ending `]` outside the loop.

Reply:
Oh right;) I first did the `,\n`and `[]`-thing but that wasn't really json-compatible, because on the last array-element it breaks json, which would've needed another exception..
I just pushed another update with the first-mentioned changes.